README.md

```diff
@@ -221,6 +223,43 @@ controller.on('message_received', processWatsonResponse);
 ```
 
 
+## Implementing event handlers
+
+Events are messages having type different than `message`.
+
+[Example](https://github.com/howdyai/botkit/blob/master/examples/facebook_bot.js) of handler:
+```js
+controller.on('facebook_postback', function(bot, message) {
+   bot.reply(message, 'Great Choice!!!! (' + message.payload + ')');
+
+});
+```
+Since they usually have no text, events aren't processed by middleware and have no watsonData attribute.
+If event handler wants to make use of some data from context, it has to read it first.
+Example:
+```js
+controller.on('facebook_postback', function(bot, message) {
+    watsonMiddleware.readContext(message.user, function(err, context) {
+        if (!context) {
+            context = {};
+        }
+        //do something useful here
+        myFunction(context.field1, context.field2, function(err, result) {
+            const newMessage = clone(message);
```

Comment:
I think `clone` should be required. Users may not know that it's the `clone` npm library